src/openforms/registrations/contrib/json_dump/config.py

```diff
@@ -70,6 +70,16 @@ class JSONDumpOptionsSerializer(JsonSchemaSerializerMixin, serializers.Serialize
         required=False,
         default=list,
     )
+    transform_to_list = serializers.ListField(
+        child=FormioVariableKeyField(),
+        label=_("Transform to list"),
+        required=False,
+        default=list,
+        help_text=_(
+            "The components which need special handling concerning the shape of the "
+            "data and needs to be transformed to a list."
+        ),
```

Comment:
```suggestion
        help_text=_(
            "Component keys in this list will be sent as an array of values rather than the default object-shape for "
            "selectboxes components."
        ),
```